powerauth-restful-model/pom.xml

```diff
@@ -24,22 +24,22 @@
 
 	<groupId>io.getlime.security</groupId>
 	<artifactId>powerauth-restful-model</artifactId>
-	<version>0.17.0</version>
+	<version>0.17.1</version>
 	<name>powerauth-restful-model</name>
 	<description>Model classes PowerAuth Standard RESTful API</description>
 
 	<parent>
 		<groupId>io.getlime.security</groupId>
 		<artifactId>powerauth-restful-integration-parent</artifactId>
-		<version>0.17.0</version>
+		<version>0.17.1</version>
 		<relativePath>../pom.xml</relativePath>
 	</parent>
 
 	<dependencies>
 		<dependency>
 			<groupId>io.getlime.core</groupId>
 			<artifactId>rest-model-base</artifactId>
-			<version>1.0.2</version>
+			<version>1.0.3</version>
```

Comment:
This is the place where the actual change occurs. See this pull request for details on what this version update means:

https://github.com/lime-company/lime-java-core/pull/2